homeassistant/components/wiser/services.yaml

```
@@ -0,0 +1,80 @@
+boost_heating:
+  description: "Set the boost mode ON defining the period of time and the desired target temperature or temperature increase delta for the boost.  Setting the temperature delta overrides any target temperature."
+  fields:
+    entity_id:
+      {
+        description: "Enter the entity_id for the room required to set the boost mode.",
+        example: "climate.wiser_lounge",
+      }
```

Please use plain YAML formatting, instead of embedding JSON.

```suggestion
    entity_id:
      description: Enter the entity_id for the room required to set the boost mode.
      example: climate.wiser_lounge
```

This needs to be applied to all other cases as well.

---

done